src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -487,14 +539,15 @@ impl A320Hydraulic {
 
     fn update_green_actuators_volume(&mut self) {
         self.green_loop
-            .update_actuator_volumes(&self.braking_circuit_norm);
-        self.braking_circuit_norm.reset_accumulators();
+            .update_actuator_volumes(&mut self.braking_circuit_norm);
     }
 
     fn update_yellow_actuators_volume(&mut self) {
         self.yellow_loop
-            .update_actuator_volumes(&self.braking_circuit_altn);
-        self.braking_circuit_altn.reset_accumulators();
+            .update_actuator_volumes(&mut self.braking_circuit_altn);
+
+        self.yellow_loop
+            .update_actuator_volumes(self.forward_cargo_door.actuator());
```

Comment:
**thought**: Is the `aft_cargo_door` missing here?

Reply:
Well, yes but it's missing in the plane too :D

Reply:
@crocket63 This confuses me, as an `aft_cargo_door` field is found in `A320Hydraulic`. Do you mean the current 3D model doesn't contain the door? If so, then I think you need to pick and choose: either add the code for that door even if it's not there at the moment, or remove all references to it. I would go for the former 😀.

Reply:
Indeed we don't have it on the model sadly, but for consistency I added it there because that will be the really easy thing to forget to add when adding the aft door...  I think code is complete for it to work on hydraulic part, we only need to wait for a animation node now

Reply:
So the aft cargo door when actually available doesn't affect the yellow loop?

Reply:
It is definitly and if you're not seeing the line I added for the aft door I don't know where the lines I wrote went :(

Reply:
Overlooked it!